tools/SetupFlow/DevHome.SetupFlow/ViewModels/SetupTargetViewModel.cs

```diff
@@ -394,6 +395,14 @@ public async Task UpdateListViewModelList(ComputeSystemsLoadedData data)
     {
         _notificationsHelper?.DisplayComputeSystemEnumerationErrors(data);
 
+        // Remove the mappings that failed to load.
+        // The errors are already handled by the notification helper.
+        foreach (var mapping in data.DevIdToComputeSystemMap.Where(map =>
+            map.Value.Result.Status == ProviderOperationStatus.Failure))

```

Comment:
nit: I think "kvp" is a more traditional name for a Key-Value pair variable. "map" sounds more like it's the whole dictionary not a KVP item from the dictionary.

Reply:
Changed